### PR TITLE
Progress Report July 2022

### PR DESCRIPTION
**Pull request by goldenx86**
Loooooong

### CODE REVIEW

File: site/content/entry/yuzu-progress-report-jul-2022/index.md [hunk 1]
```diff
@@ -0,0 +1,483 @@
...
+## Hardware section
+
+This is a new section to communicate and discuss new relevant bugs, fixes and findings related to specific hardware that can affect the user experience with yuzu.
+
+#### NVIDIA, missing the perfection that 472.12 was
+
+[We mentioned last month](https://yuzu-emu.org/entry/yuzu-progress-report-jun-2022/#psa-for-amd-radeon-users-and-nvidia-tags-along) how the 516 series of drivers is detrimental to Maxwell and Pascal users, making Vulkan unstable.
+We’re still debugging the issue, as it isn’t easy to catch, but a possible cause is suspected: GPU accelerated ASTC texture decoding.
+If you own a Maxwell or Pascal GPU, must remain on the latest driver update, and want to test if you can make Vulkan stable again, try disabling `Accelerate ASTC Texture Decoding` in `Emulation > Configure… > Graphics`.
+Please report your results in our [forums](https://community.citra-emu.org/c/yuzu-support/14) or [Discord server](https://discord.gg/u77vRWY).
```
Comment (Schplee) [inline site/content/entry/yuzu-progress-report-jul-2022/index.md:367]:
```suggestion
Please report your results on our [forums](https://community.citra-emu.org/c/yuzu-support/14) or [Discord server](https://discord.gg/u77vRWY).
```

File: site/content/entry/yuzu-progress-report-jul-2022/index.md [hunk 2]
```diff
@@ -0,0 +1,483 @@
...
+
+Another known issue caused by the 516 series of drivers is some funny flickering on trees in KOEI TECMO games like `Hyrule Warriors: Age of Calamity`.
+
+{{< single-title-imgs
+    "Day time party! To the left NVIDIA Vulkan on 516.94, to the right, 512.95 (Hyrule Warriors: Age of Calamity)"
```
Comment (Schplee) [inline site/content/entry/yuzu-progress-report-jul-2022/index.md:372]:
```suggestion
    "Day time party! NVIDIA Vulkan Left: 516.94 & Right: 512.95 (Hyrule Warriors: Age of Calamity)"
```

File: site/content/entry/yuzu-progress-report-jul-2022/index.md [hunk 3]
```diff
@@ -0,0 +1,483 @@
...
+    "./aocbug.mp4"
+    "./aoc.mp4"
+    >}}
+    
+These issues could be either regressions or undocumented behaviour changes, possible caused after following the API specification more rigorously.
```
Comment (Schplee) [inline site/content/entry/yuzu-progress-report-jul-2022/index.md:377]:
```suggestion
These issues could either be regressions or undocumented behaviour changes, possibly caused after following the API specification more rigorously.
```

File: site/content/entry/yuzu-progress-report-jul-2022/index.md [hunk 4]
```diff
@@ -0,0 +1,483 @@
...
+
+There are also performance related issues affecting users with G-SYNC/FreeSync displays, who experience low framerates (usually games get stuck at 24-30 FPS).
```
Comment (Schplee) [inline site/content/entry/yuzu-progress-report-jul-2022/index.md:379]:
```suggestion
There are also performance related issues affecting users with G-SYNC/FreeSync displays, causing low framerates (usually games get stuck at 24-30 FPS).
```

File: site/content/entry/yuzu-progress-report-jul-2022/index.md [hunk 5]
```diff
@@ -0,0 +1,483 @@
...
+We have a couple of ways to bypass this issue:
+
+1. Unfocus and refocus yuzu’s window each time you boot a game.
+2. In yuzu, disable `View > Single Window Mode`.
+3. Enable Exclusive Fullscreen from `Emulation > Configure.. > Graphics > Fullscreen Mode`. Then just play your games in fullscreen by pressing F11.
```
Comment (Schplee) [inline site/content/entry/yuzu-progress-report-jul-2022/index.md:384]:
```suggestion
3. Enable `Exclusive Fullscreen` from `Emulation > Configure.. > Graphics > Fullscreen Mode`. Then just play your games in fullscreen by pressing F11.
```

File: site/content/entry/yuzu-progress-report-jul-2022/index.md [hunk 6]
```diff
@@ -0,0 +1,483 @@
...
+
+The root of the problem is caused by some bad combination of running a Qt window inside another window, and NVIDIA’s way of detecting the framerate of windowed applications.
+Removing any of the two factors solves the low framerate while still taking advantage of Variable Refresh Rate.
+
+#### ~~AMD~~ATI OpenGL, 25 years in the making
```
Comment (Schplee) [inline site/content/entry/yuzu-progress-report-jul-2022/index.md:389]:
```suggestion
#### AMD OpenGL, 25 years in the making
```

File: site/content/entry/yuzu-progress-report-jul-2022/index.md [hunk 7]
```diff
@@ -0,0 +1,483 @@
...
+
+There are also performance related issues affecting users with G-SYNC/FreeSync displays, who experience low framerates (usually games get stuck at 24-30 FPS).
+We have a couple of ways to bypass this issue:
+
+1. Unfocus and refocus yuzu’s window each time you boot a game.
+2. In yuzu, disable `View > Single Window Mode`.
+3. Enable Exclusive Fullscreen from `Emulation > Configure.. > Graphics > Fullscreen Mode`. Then just play your games in fullscreen by pressing F11.
+
+The root of the problem is caused by some bad combination of running a Qt window inside another window, and NVIDIA’s way of detecting the framerate of windowed applications.
+Removing any of the two factors solves the low framerate while still taking advantage of Variable Refresh Rate.
```
Comment (Schplee) [inline site/content/entry/yuzu-progress-report-jul-2022/index.md:387]:
```suggestion
Removing any of the three factors solves the low framerate while still taking advantage of Variable Refresh Rate.
```

File: site/content/entry/yuzu-progress-report-jul-2022/index.md [hunk 8]
```diff
@@ -0,0 +1,483 @@
...
+{{< single-title-imgs
+    "Day time party! To the left NVIDIA Vulkan on 516.94, to the right, 512.95 (Hyrule Warriors: Age of Calamity)"
+    "./aocbug.mp4"
+    "./aoc.mp4"
+    >}}
+    
+These issues could be either regressions or undocumented behaviour changes, possible caused after following the API specification more rigorously.
+
+There are also performance related issues affecting users with G-SYNC/FreeSync displays, who experience low framerates (usually games get stuck at 24-30 FPS).
+We have a couple of ways to bypass this issue:
```
Comment (Schplee) [inline site/content/entry/yuzu-progress-report-jul-2022/index.md:380]:
```suggestion
We have a few ways to bypass this issue:
```

File: site/content/entry/yuzu-progress-report-jul-2022/index.md [hunk 9]
```diff
@@ -0,0 +1,483 @@
...
+
+The root of the problem is caused by some bad combination of running a Qt window inside another window, and NVIDIA’s way of detecting the framerate of windowed applications.
+Removing any of the two factors solves the low framerate while still taking advantage of Variable Refresh Rate.
+
+#### ~~AMD~~ATI OpenGL, 25 years in the making
+
+Hell froze over, pigs learned to fly, and starting with the Windows driver version 22.7.1, AMD introduced a completely new OpenGL driver, making Radeon cards on Windows viable options to use both APIs, not just cool kid Vulkan.
+Performance is close to 100% higher, or more, in some titles, and many rendering bugs are fixed. 
+But why write about it?
+It’s better to let the numbers do the talking:
```
Comment (Schplee) [inline site/content/entry/yuzu-progress-report-jul-2022/index.md:394]:
```suggestion
Performance is close to 100% higher, or more in some titles, and many rendering bugs are fixed. 
But why write about it, let the numbers do the talking:
```

File: site/content/entry/yuzu-progress-report-jul-2022/index.md [hunk 10]
```diff
@@ -0,0 +1,483 @@
...
+    "./mk8.png"
+    >}}
+
+We're not experts in the benchmarking area, so hopefully this helps.
+
+Here are results of an RX 6600 and a GTX 1660 SUPER running a few games in OpenGL and Vulkan. 
+22.6.1 represents the old infamous OpenGL driver, 22.7.1 is of course the new driver, Linux is represented by Mesa 22.1.3 running radeonsi with the amdgpu kernel module for OpenGL, and RADV for Vulkan, and NVIDIA is running its currently latest Windows driver.
+Remaining relevant hardware used is a 5600X and 16GB of RAM at 3600MHz, the RX 6600 was running at PCIe 4.0 8x with Smart Access Memory enabled, although that won’t make a difference, more on that later.
+Operating systems used are Windows 11 and Manjaro Linux, both up to date and on their respective default stable branches.
+yuzu is on Mainline 1112, with GPU accuracy set to normal to make GPU driver bottlenecks easier to measure, 1X resolution multiplier and Default value for Anisotropic Filtering.
```
Comment (Schplee) [inline site/content/entry/yuzu-progress-report-jul-2022/index.md:419]:
```suggestion
We're not experts in the benchmarking area, so hopefully the above graphs help.

Above are results of an RX 6600 and a GTX 1660 SUPER running a few games in OpenGL and Vulkan. 
22.6.1 represents the old infamous OpenGL driver, 22.7.1 is of course the new driver. Linux is represented by Mesa 22.1.3 running radeonsi with the amdgpu kernel module for OpenGL, and RADV for Vulkan. NVIDIA is running its latest (at the time of writing) Windows driver.
Remaining relevant hardware used is a 5600X and 16GB of RAM at 3600MHz. The RX 6600 was running at PCIe 4.0 8x with Smart Access Memory enabled, although that won’t make a difference, more on that later.
Operating systems used are Windows 11 and Manjaro Linux, both up to date and on their respective default stable branches.
yuzu is on Mainline 1112, with GPU accuracy set to normal to make GPU driver bottlenecks easier to measure, 1X resolution multiplier, and Default value for Anisotropic Filtering.
```

File: site/content/entry/yuzu-progress-report-jul-2022/index.md [hunk 11]
```diff
@@ -0,0 +1,483 @@
...
+
+A single regression under investigation and reported to AMD aside (Xenoblade Chronicles 2 crashes loading Abble’s Fountain, the measuring spot, could be caused by some driver thread crash), performance is now very close to Vulkan numbers, be it either from AMD or Nvidia.
```
Comment (Schplee) [inline site/content/entry/yuzu-progress-report-jul-2022/index.md:421]:
```suggestion
A single regression under investigation and reported to AMD aside (`Xenoblade Chronicles 2` crashes loading Abble’s Fountain, the measuring spot, could be caused by some driver thread crash), performance is now very close to Vulkan numbers, be it either from AMD or NVIDIA. 
```

File: site/content/entry/yuzu-progress-report-jul-2022/index.md [hunk 12]
```diff
@@ -0,0 +1,483 @@
...
+Operating systems used are Windows 11 and Manjaro Linux, both up to date and on their respective default stable branches.
+yuzu is on Mainline 1112, with GPU accuracy set to normal to make GPU driver bottlenecks easier to measure, 1X resolution multiplier and Default value for Anisotropic Filtering.
+
+A single regression under investigation and reported to AMD aside (Xenoblade Chronicles 2 crashes loading Abble’s Fountain, the measuring spot, could be caused by some driver thread crash), performance is now very close to Vulkan numbers, be it either from AMD or Nvidia. 
+
+It’s now perfectly valid for a Radeon user to switch to OpenGL if a specific game requires it, like for example with Xenoblade Chronicles 3, or an Unity/Unreal Engine based game (SHIN MEGAMI TENSEI V).
+As a bonus, while not being very stable, the SPIR-V shader back-end can be used on games with “simple” shaders like Super Smash Bros. Ultimate or Super Mario Odyssey, making shader building much more tolerable when compared to GLSL, giving it a performance much closer to the Nvidia-only GLASM.
```
Comment (Schplee) [inline site/content/entry/yuzu-progress-report-jul-2022/index.md:424]:
```suggestion
It’s now perfectly valid for a Radeon user to switch to OpenGL if a specific game requires it, like for example with `Xenoblade Chronicles 3`, or a Unity/Unreal Engine based game (`SHIN MEGAMI TENSEI V`).
As a bonus, while not being very stable, the SPIR-V shader back-end can be used on games with “simple” shaders like `Super Smash Bros. Ultimate` or `Super Mario Odyssey`, making shader building much more tolerable when compared to GLSL, giving it a performance much closer to the Nvidia-only GLASM.
```

File: site/content/entry/yuzu-progress-report-jul-2022/index.md [hunk 13]
```diff
@@ -0,0 +1,483 @@
...
+
+Another lesson learned from this is that some games, like Legend of Zelda: Breath of the Wild, just outright prefer Nvidia’s mature OpenGL driver. Ara ara.
```
Comment (Schplee) [inline site/content/entry/yuzu-progress-report-jul-2022/index.md:426]:
```suggestion
Another lesson learned from this is that some games, like `Legend of Zelda: Breath of the Wild`, just outright prefer NVIDIA's mature OpenGL driver. Ara ara.
```

File: site/content/entry/yuzu-progress-report-jul-2022/index.md [hunk 14]
```diff
@@ -0,0 +1,483 @@
...
+
+Lastly, to end this Red Team section.
+[In the past](https://yuzu-emu.org/entry/yuzu-progress-report-feb-2022/#vulkan-is-the-future) we reported a way to defeat RDNA2’s overcorrecting power manager in order to get decent framerates.
```
Comment (Schplee) [inline site/content/entry/yuzu-progress-report-jul-2022/index.md:429]:
```suggestion
[In the past](https://yuzu-emu.org/entry/yuzu-progress-report-feb-2022/#vulkan-is-the-future), we reported a way to defeat RDNA2’s overcorrecting power manager in order to get decent framerates.
```

File: site/content/entry/yuzu-progress-report-jul-2022/index.md [hunk 15]
```diff
@@ -0,0 +1,483 @@
...
+This method, while simple, has a downside: It’s an overclock.
+Or at least counts as one.
+
+We found an alternative that should be more globally applicable.
+The trick this time is to make the driver force high clocks on a more important section of the GPU when speaking about emulation performance in general: GPU VRAM.
+All this while keeping the warranty in check.
+
+The process is simple, make the integrated video encoder work in the background while yuzu (or any other emulator) runs.
+This is easily achieved from Radeon Software by going to `Settings > Record & Stream` and enabling `Instant Replay`.
+Intel/Linux owners should be able to reach similar results by instead using the Xbox Game Bar, or [setting OBS to keep a buffer](https://obsproject.com/forum/resources/instant-replay.575/).
```
Comment (Schplee) [inline site/content/entry/yuzu-progress-report-jul-2022/index.md:439]:
```suggestion
Intel/Linux owners should be able to reach similar results by instead using the Xbox Game Bar or [setting OBS to keep a buffer](https://obsproject.com/forum/resources/instant-replay.575/).
```

File: site/content/entry/yuzu-progress-report-jul-2022/index.md [hunk 16]
```diff
@@ -0,0 +1,483 @@
...
+
+{{< imgs
+	"./instantreplay.png| A driver toggle for power management would be so much simpler"
+  >}}
+
+After this, on yuzu enable Exclusive Fullscreen from `Emulation > Configure.. > Graphics > Fullscreen Mode`.
+Then just play your games in fullscreen by pressing F11.
+This step can be avoided if you also enable `Record Desktop`, but keep in mind this would increase your power consumption even while doing nothing.
```
Comment (Schplee) [inline site/content/entry/yuzu-progress-report-jul-2022/index.md:447]:
```suggestion
After this, in yuzu enable Exclusive Fullscreen from `Emulation > Configure.. > Graphics > Fullscreen Mode`.
Then just play your games in fullscreen by pressing F11.
This step can be avoided if you also enable `Record Desktop`, but please keep in mind this will increase your power consumption even while idling.
```

File: site/content/entry/yuzu-progress-report-jul-2022/index.md [hunk 17]
```diff
@@ -0,0 +1,483 @@
...
+
+The performance gains are the same as with the previous overclocking method, up to 73% in GPU bottlenecked titles.
+
+RX 6500 XT and RX 6400 users, since you lack a video encoder in the first place, refer to our original method mentioned at the start, or ask for a refund.
+
+#### Intel, killing support for 2 years old hardware
```
Comment (Schplee) [inline site/content/entry/yuzu-progress-report-jul-2022/index.md:453]:
```suggestion
#### Intel, killing support for 2 year old hardware
```

File: site/content/entry/yuzu-progress-report-jul-2022/index.md [hunk 18]
```diff
@@ -0,0 +1,483 @@
...
+
+Intel [recently announced](https://community.intel.com/t5/Graphics/Graphics-Driver-Support-Update-for-10th-Generation-and-Older/m-p/1403969/thread-id/108899) that their Windows driver for Gen. 9, Gen. 9.5, and Gen. 11 GPUs (that is any CPU based on the 14nm Skylake architecture and all its many marketing renames, plus Ice Lake) is now in “legacy software support”, which in layman's terms means they are officially dead.
+While this doesn’t affect yuzu immediately, any new Vulkan features we add in the future could potentially break functionality in a similar way to what happened with old AMD GCN hardware last year.
+This leaves integrated Intel GPU users with a single alternative, Linux, which offers support for even older hardware.
+For example, an ancient HD Graphics 4400 can *run* yuzu with the Mesa drivers.
```
Comment (Schplee) [inline site/content/entry/yuzu-progress-report-jul-2022/index.md:458]:
```suggestion
For example, an ancient HD Graphics 4400 *can* run yuzu with the Mesa drivers.
```

File: site/content/entry/yuzu-progress-report-jul-2022/index.md [hunk 19]
```diff
@@ -0,0 +1,483 @@
...
+
+Users should consider learning Linux if a hardware upgrade is not a viable option in the near future, Mesa has always offered better performance for Intel GPUs.
```
Comment (Schplee) [inline site/content/entry/yuzu-progress-report-jul-2022/index.md:460]:
```suggestion
Users should consider learning how to use Linux if a hardware upgrade is not a viable option in the near future, Mesa has always offered better performance for Intel GPUs.
```

File: site/content/entry/yuzu-progress-report-jul-2022/index.md [hunk 20]
```diff
@@ -0,0 +1,483 @@
...
+
+## Future projects
+
+Part 2 of `Project Y.F.C.` is a bit delayed for now, real life issues as previously mentioned, but its feature list and expected progression is laid out.
+
+`Project London` is progressing in a healthy fashion, we loved the internal testing done so far.
+
+And a possibility has just recently started to open for even better GPU performance in the (not so near) future. 
+`GPU fastmem` is one of the features that Rodrigo had to leave for later, before passing the torch and moving onto “greener sides”.
```
Comment (Schplee) [inline site/content/entry/yuzu-progress-report-jul-2022/index.md:469]:
```suggestion
And a possibility has just recently started to open for even better GPU performance in the (not so near) future. 
`GPU fastmem` is one of the features that [Rodrigo ](https://github.com/ReinUsesLisp) had to leave for later, before passing the torch and moving onto “greener sides”.
```

File: site/content/entry/yuzu-progress-report-jul-2022/index.md [hunk 21]
```diff
@@ -0,0 +1,483 @@
...
+
+The main roadblock holding GPU fastmem was driver support, which is now a mostly solved issue, we *only* need to start talks with the AMD, Intel, and the AMD Linux kernel module developers to ask for some increased limits.
+Once those obstacles are out of the way, yuzu should for example be able to take partial advantage of Resizable BAR/Smart Access Memory, helping reduce PCIe bottlenecks, and should help improve particle rendering to make GPU accuracy a less critical performance setting.
+No pressure, Blinkhawk!
```
Comment (Schplee) [inline site/content/entry/yuzu-progress-report-jul-2022/index.md:473]:
```suggestion
The main roadblock holding `GPU fastmem` back was driver support, which is now a mostly solved issue. We *only* need to start talks with the AMD, Intel, and the AMD Linux kernel module developers to ask for some increased limits.
Once those obstacles are out of the way, yuzu should, for example, be able to take partial advantage of Resizable BAR/Smart Access Memory, helping reduce PCIe bottlenecks, and should help improve particle rendering to make GPU accuracy a less critical performance setting.
No pressure, Blinkhawk!
```

File: site/content/entry/yuzu-progress-report-jul-2022/index.md [hunk 22]
```diff
@@ -0,0 +1,483 @@
...
+
+That’s all folks! This one turned out to be longer than expected. Thank you for staying until the end, and we hope to see you again next month!
+Thank you NazD for the summary pic!
+
+~~Movistar, you can go to hell.~~
```
Comment (Schplee) [inline site/content/entry/yuzu-progress-report-jul-2022/index.md:478]:
```suggestion
```

**Review by Schplee — CHANGES_REQUESTED** [review]

File: site/content/entry/yuzu-progress-report-jul-2022/index.md [hunk 23]
```diff
@@ -0,0 +1,483 @@
...
+	"./ir.png| "
+  >}}
+
+Users wanting to play with this setting can find it in `Emulation > Configure… > Controls > Advanced tab > Infrared Camera`.
+
+{{< imgs
+	"./camera.png| "
+  >}}
+
+This work doesn’t include the moment processor required by `1-2-Switch!` yet
```
[the remark this answers is not in the document]
Comment (goldenx86) [reply site/content/entry/yuzu-progress-report-jul-2022/index.md:303]: Asked it, it's moment.

File: site/content/entry/yuzu-progress-report-jul-2022/index.md [hunk 24]
```diff
@@ -0,0 +1,480 @@
...
+
+## Input improvements
+
+A unique feature of the Nintendo Switch is the capability to use infrared cameras installed in the right Joy-Con.
+The main function of the cameras is to detect shapes and measure the distance to objects, but it can also be used to transmit a feed to a screen, letting you turn your Joy-Con into a heat-seeking monstrosity. [Fox-2](https://en.wikipedia.org/wiki/Fox_(code_word))!
+
+Interested in adding this awesome feature to yuzu, and providing full support for games like `Game Builder Garage` or the `Nintendo Labo` collection, [german77](https://github.com/german77) {{< gh-hovercard "8636" "emulated the clustering processor" >}} required to let the games access the camera on the Joy-Cons or any camera the user wants to provide, even if it is a desktop capture obtained from OBS Studio.
+
+{{< imgs
+	"./ir.png| While this looks like a homebrew app, it's in fact the real configuration Nintendo provides on its games"
```
Comment (Schplee) [inline site/content/entry/yuzu-progress-report-jul-2022/index.md:294]:
```suggestion
	"./ir.png| While this looks like a homebrew app, it's in fact the real IR interface Nintendo provides for its games"
```

File: site/content/entry/yuzu-progress-report-jul-2022/index.md [hunk 25]
```diff
@@ -0,0 +1,480 @@
...
+  >}}
+
+Users wanting to play with this setting can find it in `Emulation > Configure… > Controls > Advanced tab > Infrared Camera`.
+
+{{< imgs
+	"./camera.png| You can find the setting here"
```
Comment (Schplee) [inline site/content/entry/yuzu-progress-report-jul-2022/index.md:300]:
```suggestion
	"./camera.png| What a sleek little menu"
```

File: site/content/entry/yuzu-progress-report-jul-2022/index.md [hunk 26]
```diff
@@ -0,0 +1,480 @@
...
+We have a few ways to bypass this issue:
+
+1. Unfocus and refocus yuzu’s window each time you boot a game.
+2. In yuzu, disable `View > Single Window Mode`.
+3. Enable `Exclusive Fullscreen` from `Emulation > Configure.. > Graphics > Fullscreen Mode`. Then just play your games in fullscreen by pressing F11.
+
+The root of the problem is caused by some bad combination of running a Qt window inside another window, and NVIDIA’s way of detecting the framerate of windowed applications.
+Removing any of the two factors solves the low framerate while still taking advantage of Variable Refresh Rate.
+
+#### ~~ATI~~AMD OpenGL, 25 years in the making
```
Comment (Schplee) [inline site/content/entry/yuzu-progress-report-jul-2022/index.md:388]:
```suggestion
#### AMD OpenGL, 25 years in the making
```

File: site/content/entry/yuzu-progress-report-jul-2022/index.md [hunk 27]
```diff
@@ -0,0 +1,480 @@
...
+	"./ir.png| While this looks like a homebrew app, it's in fact the real configuration Nintendo provides on its games"
+  >}}
+
+Users wanting to play with this setting can find it in `Emulation > Configure… > Controls > Advanced tab > Infrared Camera`.
+
+{{< imgs
+	"./camera.png| You can find the setting here"
+  >}}
+
+This work doesn’t include the moment processor required by `1-2-Switch!` yet
```
Comment (Schplee) [inline site/content/entry/yuzu-progress-report-jul-2022/index.md:303]:
```suggestion
This work doesn’t include the moment processor required by `1-2-Switch!` just yet.
```

File: site/content/entry/yuzu-progress-report-jul-2022/index.md [hunk 28]
```diff
@@ -0,0 +1,480 @@
...
+
+Hell froze over, pigs learned to fly, and starting with the Windows driver version 22.7.1, AMD introduced a completely new OpenGL driver, making Radeon cards on Windows viable options to use both APIs, not just cool kid Vulkan.
+Performance is close to 100% higher, or more in some titles, and many rendering bugs are fixed. 
+But why write about it, let the numbers do the talking:
+
+{{< imgs
+	"./perf.png| Thank you RodrigoTR!"
+  >}}
+
+Tch, okey, that's a lot of numbers, let's try to make it easier to digest.
```
Comment (Schplee) [inline site/content/entry/yuzu-progress-report-jul-2022/index.md:398]:
```suggestion
Wow! That's a lot of numbers, let's try to make it easier to digest:
```

**Review by Schplee — CHANGES_REQUESTED** [review]

**Comment by CeruleanSky** [comment]
If possible and if it isn't too late, could a link to a ticket that best tracks the Nvidia 516.xx problem be added to the section discussing it?  I created a pastebin(as not to bump bunches of tickets) of 516 or pascal nvidia candidates on the Yuzu issue tracker to help in this process, but I don't know which one best represents the issue or if a better one exists.  The 516 driver seems to be a requirement for Spiderman Remastered so a lot more people might be running into it.


https://pastebin.com/5iUZaFPA

**Comment by goldenx86** [comment]
Sorry, I don't want to track issues in the progress reports. Issues get outdated pretty quickly, and a monthly report is not ideal for tracking them.
Github is preferred for this.
